src/app/app.module.ts

```diff
@@ -24,9 +27,11 @@ import { initializer } from '_utils/app-init';
 
 @NgModule({
   declarations: [
-    AppComponent,
+    AppComponent, CreateGroupDialog,
```

Comment:
does CreateGroupDialog need to be declared here for it to work? I don't think it _should_.

Reply:
I got the error: "Component CreateGroupDialog is not part of any NgModule or the module has not been imported into your module" when I removed CreateGroupDialog. Not sure why it is required though, maybe I have imports wrong somewhere else?